src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -47,33 +49,24 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
-        Query groupQuery = new Query(GROUP_KIND);
-        PreparedQuery preparedGroupQuery = datastore.prepare(groupQuery);
-
-        ImmutableList.Builder<ImmutableMap<String, String>> groupsListBuilder = new ImmutableList.Builder<>();
-        for (Entity group : preparedGroupQuery.asIterable()) {
-            if (defaultGroup == null) {
-                defaultGroup = group.getKey().getName();
-            }
-
-            ImmutableMap<String, String> groupMap = ImmutableMap.of(
-                GROUP_NAME_PROPERTY, (String) group.getProperty(GROUP_NAME_PROPERTY),
-                GROUP_ID_PROPERTY, group.getKey().getName());
-            groupsListBuilder.add(groupMap);
-        }
-        ImmutableList<ImmutableMap<String, String>> groupsList = groupsListBuilder.build();
+        // ImmutableList<ImmutableMap<String, String>> groupsList = 
+        //     ServletUtils.getGroupsList(request.getUserPrincipal().getName());
```

Comment:
Remove these if your not going to use them. Unless your using the sample in the next lines